empress/support_files/templates/side-panel.html

```diff
@@ -171,6 +171,6 @@
     <button id="center-tree-btn">Reset Camera</button>
   </p>
   <p class="side-panel-notes indented">
-    Centers the screen on the root node of the tree.
+    Centers the screen at the average coordinates of the tree's nodes.
```

Comment:
Do you think this would make sense:

```suggestion
    Repositions the camera at the center of the tree.
```

Trying to think of something less technical but still accurate.

Reply:
Sure, sounds good to me.

Reply:
I think users might ask in the future for a "recenter camera" button that fits the screen to show the *entire* tree... but we can tackle that in the future.